data/Templates/Hl7v2/ID/_Encounter.liquid

```diff
@@ -1,5 +1,6 @@
 {% capture identifiers -%}
-{% if PV1.19.1 and PV1.19.4 -%}{{PV1.19.1.Value}},{{PV1.19.4.Value}}
+{% if PV1.3.1 and PV1.19.1 and PV1.19.4 -%}{{PV1.3.1.Value}}{{PV1.19.1.Value}},{{PV1.19.4.Value}}
```

Comment:
We've updated the format of ID templates in `dotliquid` branch, you can update your branch.